book/scale_nodes.rst

```diff
@@ -0,0 +1,360 @@
+
+- **Maintenance mode**: there shouldn't be any problems with operations that require the maintenance mode, as default 
+  configuration settings place the 'maintenance_lock' file in the cache directory, which is already shared among the 
+  web-nodes.
+  If you want to store the lock file in another location, see the `documentation on the 
+  LexicMaintenanceBundle <https://github.com/lexik/LexikMaintenanceBundle/blob/master/Resources/doc/index.md>`_.
+
+- **CRON** jobs: there shouldn't be any problems, as the queue is stored in the Database, so all the web-nodes will 
+  know the actual state of each job. 
+  By default the job daemon is configured to run 5 concurrent (simultaneous) jobs. So, for example, if we have 5 web 
+  farm nodes we can decrease this value to 1. In this case, each node will run only 1 CRON job. In some cases it can 
+  also be more efficient to have an independent node just for CRON jobs.
+
+- **REDIS**: use `SncRedisBundle <https://github.com/snc/SncRedisBundle>`_
+  
+  - was tested with: the version 1.1.9 
+
+  - was tested with PhpRedis extension, however you may also use Predis (but only versions 0.8.* are supported by 
+    the SncRedisBundle).
+
+    - The PhpRedis extension file (.so) can be installed via the package manager
+
+      - for Linux: yum/dnf/apt-get install php-pecl-redis
+      
+      - for MacOs via homebrew: brew install php55-redis
+
+    - or compiled from the sources - see https://github.com/phpredis/phpredis
+
+If you don't want to use the extension, you can install the `Predis <https://github.com/nrk/predis>`_ bundle.
+
+- **APC** (http://php.net/manual/en/book.apc.php, https://pecl.php.net/package/APCu, https://github.com/krakjoe/apcu) 
+  can be installed via package manager in the same way as PhpRedis .
+
+.. note::
+
+    For APCu installation with the version **4.0.7** there appears a `key inconsistency bug <key inconsistency bug>`_, 
+    which can lead to php notices or exceptions when rendering attachments of the image type. In order to solve the 
+    issue, install an older version or compile the extension from the master branch.
+
+
+CONFIGURATION:
+--------------
+
+**All the configuration changes can be done in [Application root]/app/config/{config.yml, security.yml} or inside your 
+own bundle in the app.yml file.**
+
+1. **add** SncRedisBundle into composer.json
+
+.. code-block:: yaml
+
+   "require": {
+       ...
+       "snc/redis-bundle": "1.1.*",
+       "predis/predis": "0.8.7" #in case using Predis instead of PhpRedis
+   }
+      
+2. **execute** the "composer update"
+
+3. **register** the SncRedisBundle 
+
+  - in AppKernel.php
+
+.. code-block:: php
+
+   ...
+   public function registerBundles()
+   {
+       $bundles = array(
+       //bundles
+           new Snc\RedisBundle\SncRedisBundle()
+       );
+   ...
+
+  - or in the bundles.yml (see the example) 
+   [Application root]/src/Oro/src/Oro/Bundle/PlatformBundle/Resources/config/oro/bundles.yml
```

Comment:
Examples from documentation usually considered as recommendations, i would avoid potentially not correct examples